MELA/src/TUtil.cc

```diff
@@ -5355,26 +5350,32 @@ double TUtil::MadgraphMatEl(
       << "\trenQ: " << renQ << " ( x " << event_scales->ren_scale_factor << "), facQ: " << facQ << " ( x " << event_scales->fac_scale_factor << ")\n"
       << "\tAfter set, alphas scale: " << scale_.scale << ", PDF scale: " << facscale_.facscale << ", alphas(Qren): " << alphasVal << ", alphas(MZ): " << alphasmzVal << endl;
   }
-  int procid = -1;
-  double scale2 = 1; //This is useless for ggH calculations
+  int procid = -1; //always set to -1 just cause
+  double scale2 = 1; //This is useless for calculations
   if (verbosity>=TVar::DEBUG_VERBOSE){
-    MELAout << "Input vectors to MADGRAPH function in order as id, px, py, pz, E:" << endl;
-    for(int i = 0; i < nPDG; i++){
-      MELAout << "id of " << pdgs[i] << " & vector of " << p[i][1] << ", " << p[i][2] << ", " << p[i][3] << ", " << p[i][0] << endl;
-    }
+    // MELAout << "Input vectors to MADGRAPH function in order as id, px, py, pz, E:" << endl;
+    // for(int i = 0; i < nPDG; i++){
+    //   MELAout << "id of " << pdgs[i] << " & vector of " << p[i][1] << ", " << p[i][2] << ", " << p[i][3] << ", " << p[i][0] << endl;
+    // }
     MELAout << "Raw Input to FORTRAN:" << endl;
     for(int i = 0; i < nPDG; i++){
       for(int j = 0; j < 4; j++){
-        MELAout << p_for_fortran[i*4+j] << " ";
+        MELAout << p[i*4+j] << " ";
       }
       MELAout << endl;
     }
   }
-  madMela::update_all_coup_();
+  madMela::update_all_coup(process, production);
```

Comment:
`madMela::update_all_coup` now requires the process and the production to correctly compute what it needs